bookomate/templates/allauth/signup.html

```diff
@@ -6,7 +6,12 @@
 {% block head_title %}{% trans "Signup" %}{% endblock %}
 
 {% block content %}
-
+<script src="https://www.google.com/recaptcha/api.js"></script>
```

Comment:
By convention you should load scripts after the body of the HTML.

Reply:
Thanks for telling me that